schema/schema.go

```diff
@@ -73,3 +79,26 @@ type Ignored struct {
 	Exclusion  bool
 	ForeignKey bool
 }
+
+func PrintSourceType(ty Type) string {
```

Comment:
I think this should would be clearer using a receiver type.
Also, given this function is in the schema package, we can shorten the function name.

func (ty *Type) Print() {
  ...
}

Reply:
I have converted to use receiver type and changed function name to Print(). Please merge this PR, so then I will send `mysqldump` PR